Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificates.kt

```diff
@@ -15,10 +15,11 @@ data class PersonCertificates(
         certificates.findHighestPriorityCertificate()
     }
 
-    // TODO: would be implemented in admission state logic task
-    // null when certificates list is empty
+    val validCertificates: List<CwaCovidCertificate>
+        get() = certificates.filter { it.isValid }
```

Comment:
`isValid` is dealing with expired Test Certificate as Valid, I think this is not what is required here, All certs should be either `Valid or Expiring Soon` to be eligible for G rules

Reply:
You are absolutely right. Good catch! Adjusted it so that test certificates with state `ExpiringSoon` are also considered for the admission state. Added tests for this too.